deployments/sgx_plugin/overlays/epc-register/add-epc-register-initcontainer.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: intel-sgx-plugin
+spec:
+  template:
+    spec:
+      serviceAccountName: sgx-epc-extres-api
+      initContainers:
+      - name: sgx-epc-extended-resource
+        image: intel/intel-sgx-initcontainer:devel
+        imagePullPolicy: IfNotPresent
+        args:
+          - -register
```

Comment:
@dmitsh the image entrypoint is not the binary but a script that copies the binary to NFDs source hooks dir. Therefore, `command` is needed here, I think.

Reply:
addressed